### DISCUSSION TITLE
Why I cannot bind m3 to open Edge browser?

### DISCUSSION THREAD

**Discussion by fakunyang** [seerge/g-helper#874] [Q&A]
I copy the address "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe" in the action text field(with quotes or without quotes), but no response after I press the M3. And other apps in D disc or in the root directory of the C disc can open successfully.

**Comment by seerge** [comment]
@fakunyang try this build

[GHelper.zip](https://github.com/seerge/g-helper/files/12145766/GHelper.zip)

**Reply by fakunyang** [reply]
Thanks very much! Now it works smoothly.
